### PR TITLE
docs fixes

### PR DESCRIPTION
**Pull request by ebuchman**
Some minor fixes and filled out the TEE doc a bit

### CODE REVIEW

File: docs/getting_started.md [hunk 1]
```diff
@@ -7,7 +7,7 @@
 - [Simple Example](#simple-example)
 - [Installation](#installation)
 - [Local Testnet without SGX](#local-neutrond-testnet-without-sgx)
-- [Real Testnet with SGX](#hard-example---real-testnet-with-azure-sgx))
```
Comment (davekaj) [inline docs/getting_started.md:10]: will have to update the link to the changed title too

File: docs/tees.md [hunk 2]
```diff
@@ -1,10 +1,48 @@
...
+
+## Resources on TEE security.
+
+For a great technical background on SGX, see [Intel SGX
+Explained](https://eprint.iacr.org/2016/086.pdf).
+
+This paper contains an infamous quote:
+
+> our security analysis reveals that the limitations in SGX’s guarantees mean that a security conscious software developer cannot in good conscience
+rely on SGX for secure remote computation
```
Comment (davekaj) [inline docs/tees.md:30]: not sure if this quote was meant to be on 2 lines

**Review by davekaj — APPROVED** [review]
2 small comments

File: docs/getting_started.md [hunk 3]
```diff
@@ -7,7 +7,7 @@
...
+- [Real Testnet with SGX](#hard-example---real-testnet-with-azure-sgx)
```
Comment (ebuchman) [inline docs/getting_started.md:10]:
```suggestion
- [Real Testnet with SGX](#real-testnet-with-azure-sgx)
```

File: docs/tees.md (hunk 2 above)
Comment (ebuchman) [reply docs/tees.md:30]: yep!

**Pull request merged**